SpatialGDK/Source/Public/Schema/StandardLibrary.h

```diff
@@ -37,77 +47,16 @@ struct Coordinates
 	}
 };
 
-const Worker_ComponentId POSITION_COMPONENT_ID = 54;
-
-struct SpatialPosition : SpatialComponent
-{
-	static const Worker_ComponentId ComponentId = POSITION_COMPONENT_ID;
-
-	SpatialPosition() = default;
-
-	SpatialPosition(const Coordinates& InCoords)
-		: Coords(InCoords) {}
-
-	SpatialPosition(const Worker_ComponentData& Data)
-	{
-		Schema_Object* ComponentObject = Schema_GetComponentDataFields(Data.schema_type);
-
-		Schema_Object* CoordsObject = Schema_GetObject(ComponentObject, 1);
-
-		Coords.X = Schema_GetDouble(CoordsObject, 1);
-		Coords.Y = Schema_GetDouble(CoordsObject, 2);
-		Coords.Z = Schema_GetDouble(CoordsObject, 3);
-	}
-
-	Worker_ComponentData CreatePositionData()
-	{
-		Worker_ComponentData Data = {};
-		Data.component_id = POSITION_COMPONENT_ID;
-		Data.schema_type = Schema_CreateComponentData(POSITION_COMPONENT_ID);
-		Schema_Object* ComponentObject = Schema_GetComponentDataFields(Data.schema_type);
-
-		Schema_Object* CoordsObject = Schema_AddObject(ComponentObject, 1);
-
-		Schema_AddDouble(CoordsObject, 1, Coords.X);
-		Schema_AddDouble(CoordsObject, 2, Coords.Y);
-		Schema_AddDouble(CoordsObject, 3, Coords.Z);
-
-		return Data;
-	}
-
-	static Worker_ComponentUpdate CreatePositionUpdate(const Coordinates& Coords)
-	{
-		Worker_ComponentUpdate ComponentUpdate = {};
-		ComponentUpdate.component_id = POSITION_COMPONENT_ID;
-		ComponentUpdate.schema_type = Schema_CreateComponentUpdate(POSITION_COMPONENT_ID);
-		Schema_Object* ComponentObject = Schema_GetComponentUpdateFields(ComponentUpdate.schema_type);
-
-		Schema_Object* CoordsObject = Schema_AddObject(ComponentObject, 1);
-
-		Schema_AddDouble(CoordsObject, 1, Coords.X);
-		Schema_AddDouble(CoordsObject, 2, Coords.Y);
-		Schema_AddDouble(CoordsObject, 3, Coords.Z);
-
-		return ComponentUpdate;
-	}
-
-	Coordinates Coords;
-};
-
-using WriteAclMap = TMap<Worker_ComponentId, WorkerRequirementSet>;
-
-const Worker_ComponentId ENTITY_ACL_COMPONENT_ID = 50;
-
-struct SpatialEntityAcl : SpatialComponent
+struct EntityAcl : improbable::Component
```

Comment:
If we're already in the namespace of `improbable` do we need the `improbable::Component` in this context?

Reply:
Same applies to the other `Component`s in this class :D